code/modules/spells/spell_types/conjure/simian.dm

```diff
@@ -13,6 +13,9 @@
 	invocation = "OOGA OOGA OOGA!!!!"
 	invocation_type = INVOCATION_SHOUT
 
+	///Our gorilla transformation spell, additionally granted to the user at max level.
+	var/datum/action/cooldown/spell/shapeshift/gorilla/gorilla_transformation
```

Comment:
make sure this is deleted on `Destroy()`.

Reply:
ah hell now its me who skipped _your_ review

Reply:
What a world